ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -366,6 +367,31 @@ export default function BuildQuote({
             defaultToAll
           />
         </div>
+        {selectedToToken?.address &&
+          selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address && (
+            <div className="build-quote__token-message">
+              <span>
+                {t('verifyThisTokenOn', [
+                  <span
```

Comment:
We should use a `button` or `a` or something the browser recognizes is clickable, so that it has more accessible behaviour by default. We shouldn't be using `span`s and `div`s for clickable things